runtime/gc_glue_java/ScavengerDelegate.cpp

```diff
@@ -714,6 +733,30 @@ MM_ScavengerDelegate::switchConcurrentForThread(MM_EnvironmentBase *env)
 #endif
     }
 }
+
+void
+MM_ScavengerDelegate::signalThreadsToFlushCaches(MM_EnvironmentBase *envBase)
+{
+	J9VMThread *walkThread = NULL;
+
+	GC_VMThreadListIterator vmThreadListIterator(_javaVM);
+
+	GC_VMInterface::lockVMThreadList(_extensions);
+
+	while((walkThread = vmThreadListIterator.nextVMThread()) != NULL) {
+		_javaVM->internalVMFunctions->J9SignalAsyncEvent(_javaVM, walkThread, _flushCachesAsyncCallbackKey);
```

Comment:
Is it worth fetching the `J9InternalVMFunctions` table into a local outside the loop?